src/chapters/ldap-auth.xml

```diff
@@ -384,6 +384,15 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                                 have no effect on Guacamole configurations.</emphasis></para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-user-search-filter</property></term>
+                    <listitem>
+                        <para>The search filter used to query the LDAP tree for users that
+                            can log into and be granted privileges in Guacamole.  <emphasis>If
+                            this property is omitted the default of (objectClass=*) will be used.
```

Comment:
Please place `(objectClass=*)` within quotes, so it's clear that it's a literal value.